bundles/org.openhab.binding.kodi/src/main/java/org/openhab/binding/kodi/internal/protocol/KodiConnection.java

```diff
@@ -68,6 +68,7 @@
     private static final String PROPERTY_FANART = "fanart";
     private static final String PROPERTY_THUMBNAIL = "thumbnail";
     private static final String PROPERTY_VERSION = "version";
+    private static final String PROPERTY_SCREENSAVER = "System.ScreensaverActive";
```

Comment:
Why using this new kind of naming for property?

Reply:
That's simply how Kodi is calling it.

Reply:
It is not a thing property?

Reply:
No, all these `PROPERTY_*` are properties of the JSON that is sent to/from Kodi.

Reply:
Ok, thank you